src/pages/charter.js

```diff
@@ -154,31 +154,30 @@ const CharterPage = ({ location: {pathname} }) => {
-          <li>One representative (each a “eBPF Runtime Representative”) for each additional
-          open-source eBPF runtime implementation. Each eBPF Runtime Representative will
-          be nominated and elected by the applicable oversight body of each eBPF runtime,
-          with nominations and elections subject to approval by the BSC. Nominees need to be
-          active contributors to the applicable eBPF runtime.</li>
-          <li>Up to two representatives (“Additional Project Representatives”) from additional
-          open source projects that are important to the eBPF ecosystem, as determined by the
-          BSC. Each Additional Project Representative need to be an active contributor to one
-          of these additional projects. The Additional Project Representatives will not be
-          employed by the same Company or by Related Companies. Each Additional Project
-          Representative must be nominated and elected by the BSC.</li>
-          <li>Up to three representatives with maintainer status (“Maintainer Representatives”) of
-          major eBPF Foundation projects as approved by the BSC. Maintainer Representatives
-          are nominated and elected by the applicable oversight body of each major eBPF
-          Foundation project, with nominations and elections subject to approval by the BSC.
-          No more than one Maintainer Representative may be employed by any one company
-          or group of Related Companies.</li>
+          <li>Two Linux Kernel representatives from the group of eBPF Linux
+          kernel maintainers (as specified in the Linux eBPF maintainer file). The Linux Kernel representatives
+          must not be employed by the same Company or by Related Companies.</li>
+
+          <li>Up to two representatives from other open source eBPF runtime implementations.
+          Each “eBPF Runtime Representative” will represent a different eBPF runtime chosen
+          by the BSC, and must be an active contributor to that eBPF runtime.</li>
+
+          <li>Up to three Technical Project representatives. Each Technical Project
+          Representative candidate must be an active contributor to a different Technical Project
+          chosen by the BSC.</li>
+
+          <li>Up to two eBPF Landscape Project representatives. Each eBPF Landscape Project Representative
+          candidate must be an active contributor to a different eBPF Landscape Project chosen by the BSC.</li>
         </ol>
         </li>
       <p>There can be at most two representatives on the BSC employed by the same company or by
       Related Companies. While normally members will serve two-year terms, the BSC will set
       initial term length so as to stagger elections.</p>  
+        <p>Representatives from each project (Linux kernel, other runtimes, etc.) are nominated
+           and elected by the applicable oversight body (such as the set of eBPF maintainers),
```

Comment:
In a similar vein to what I raised in the BSC meeting: as a "Landscape Project" representative this seems really heavy weight. We don't have a formal oversight body and no method of elections.

Since the BSC ultimately has to ratify (aka elect) members, why not have the BSC directly nominate candidates? Seems like we're doubling the election load here.

If that is not palatable I would much prefer if we could tone down the language here. See below:

```suggestion
           by the applicable oversight body of the project, such as the set of maintainers or similar.
```